qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -13,8 +13,9 @@
 import tempfile
 import traceback
 
+from . import util
+from . import coverage
 from authproxy import AuthServiceProxy, JSONRPCException
-from util import *
```

Comment:
Indeed; explicitly importing symbols reduces namespace pollution and slightly reduces chances for odd bugs.